### PR TITLE
Network/Util: Extract network util functions from MainService

### PR DESCRIPTION
**Pull request by niley7464**
This patch extracts network realted util functions from MainService.

### CODE REVIEW

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/network/Util.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import kotlin.concurrent.thread
+
+const val TAG = "NetworkUtil"
+
+/**
+ * Returns the IP address based on whether the app is running on an emulator or not.
+ *
+ * @param isRunningOnEmulator Indicates whether the app is running on an emulator.
+ * @return The IP address as a string.
+ * @todo Add an ApplicationContext Parameter.
```
Comment (wooksong) [inline ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/network/Util.kt:18]: Why?

Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/network/Util.kt:18]: I copied the part from the original comment, is it okay if I delete it?

Comment (wooksong) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/network/Util.kt:18]: Sure!

**Review by wooksong — APPROVED** [review]
LGTM!

**Pull request merged**